test/helpers/password.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 // The password used for integration tests:
-let password = 'CozyTest$1'
+let password = process.env.COZY_PASSPHRASE || 'CozyTest_1'
```

Comment:
Or do you think of some scenario where still being able to overwrite it is more important?